data/json/items/crossbows.json

```diff
@@ -249,19 +249,20 @@
     "color": "green",
     "name": "pistol crossbow",
     "description": "A small concealable pistol-like crossbow.  It's weak due to its small size and draw, so it is best suited for hunting small game.  Bolts fired from this weapon have a good chance of remaining intact for re-use.",
-    "price": 55000,
-    "material": [ "steel", "wood" ],
+    "price": 5500,
+    "material": [ "steel", "plastic" ],
     "flags": "PRIMITIVE_RANGED_WEAPON",
     "skill": "pistol",
     "ammo": "bolt",
-    "weight": 900,
+    "weight": 9000,
```

Comment:
I'm finding 80-lb. draw weight pistol crossbows that weigh 2 lb/.9 kg. Where are you 9 kg pistol crossbows>

Reply:
That was a mistype, I think that was supposed to be price?